packages/design-system/package.json

```diff
@@ -14,8 +14,8 @@
   "types": "dist/types/index.d.ts",
   "files": [
     "dist",
-    "src/components/**/*.jsx",
-    "!src/components/**/*.test.jsx"
+    "src/components/**/*.?sx",
+    "!src/components/**/*.test.?sx"
```

Comment:
We could probably improve this line as well, this is taken from some code in our scripts package. we generally just want to only include src component files so that we can extract prop docs
```
    "!src/**/*{.test,.spec,.d}.{jsx,tsx}",
```